docs/sources/experiments/running.md

```diff
@@ -10,92 +10,163 @@ To use a Triage experiment, you first need:
-        config=experiment_config,
-        db_engine=triage.create_engine(...),
-        model_storage_class=FSModelStorageEngine,
-        project_path='/path/to/directory/to/save/data'
-    )
+
+If you have the database information stored somewhere else, you may pass it to the top-level 'triage' command:
+
+```bash
+triage -d mydbconfig.yaml experiment -c example_experiment_config.yaml --project-path '/path/to/directory/to/save/data'
 ```
 
-These lines are a bit dense: what is happening here?
+### Python
 
-- `SingleThreadedExperiment`:  There are different Experiment classes available in ``triage.experiments`` to use, and they each represent a different way of executing the experiment, which we'll talk about in more detail later. The simplest (but slowest) is the `SingleThreadedExperiment`.
-- `config=experiment_config`: The bulk of the work needed in designing an experiment will be in creating this experiment configuration. See [defining.md](Defining an Experiment); more detailed instructions on each section are located in the example file. Generally these would be easiest to store as a file (or multiple files that you construct together) like that YAML file, but the configuration is passed in dict format to the Experiment constructor and you can store it however you wish.
-- `db_engine=triage.create_engine(...)`: A SQLAlchemy database engine. This will be used both for querying your source tables and writing results metadata. To create this, we recommend using `triage.create_engine`. It will create a serializable version of the engine that will be fully reconstructed in multiprocess contexts. If you pass a regular SQLAlchemy engine, in these contexts the engine will be reconstructed with the *URL only*, which may cancel other settings you have used to configure your engine.
-- `model_storage_class=FSModelStorageEngine`: The path to a model storage engine class. A model storage engine class is one that wraps all model storage I/O operations, allowing storage medium (e.g. filesystem, S3) to be switched easily. Triage provides a few of these in [triage.component.catwalk.storage](../src/triage/component/catwalk/storage.py). Shown here is the `FSModelStorageEngine`, which is a good place to start.
-- `project_path='/path/to/directory/to/save/data'`: The path to where you would like to store design matrices and trained models. May be an s3 path (e.g. s3://bucket-name/project-directory), in which case s3 will be used to store matrices and models.
+When running an experiment in Python, the database information is passed in the form of a SQLAlchemy database engine, and the experiment information is passed as a dictionary rather as YAML specifically.
 
-With that in mind, a more full version of the experiment instantiation might look like this
+```python
+from triage.experiments import SingleThreadedExperiment
 
+experiment = SingleThreadedExperiment(
+    config=experiment_config, # a dictionary
+    db_engine=create_engine(...),
```

Comment:
Yeah I wasn't sure how to message the part about the triage create_engine. We can push it in general, because in the future it might also have cool features (like role management) that are relevant outside of the multicore context.